.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+<!--
```

Comment:
nit: (so that when GitHub autofills the PR body from the commit message, it doesn't become "thecomment<!--")

```suggestion

<!--
```